### PR TITLE
Minor fixes to testing readme files

### PR DESCRIPTION
**Pull request by jukkar**
Very minor fixes to various testing related readme files.

### CODE REVIEW

**Review by fmuyassarov — APPROVED** [review]
LGTM

File: test/e2e/README.md [hunk 1]
```diff
@@ -6,44 +6,44 @@ Before running E2E tests ensure that you have all the required components locall
...
 
 3. Build containerd binaries that include NRI support (minimum tag version [`containerd 1.7.0-beta.1`](https://github.com/containerd/containerd/releases/tag/v1.7.0-beta.1) or +)
 
     ```shell
-    $ git clone https://github.com/containerd/containerd.git
-    $ cd containerd
-    $ make build
+    git clone https://github.com/containerd/containerd.git
+    cd containerd
+    make build
```
Comment (fmuyassarov) [inline test/e2e/README.md:23]: btw, not related but just remembered. `make build` should be replaced by `make`. Make build is not generating binaries we need in the e2e.

Comment (klihub) [reply test/e2e/README.md:23]: @fmuyassarov Hmm, how is that not related ? AFAICT, @jukkar is touching exactly the line where this should then be fixed. Or did I misunderstand your comment ?

Comment (jukkar) [reply test/e2e/README.md:23]: I tested this and indeed "make build" is not building binaries. I do not know from where I have taken the commands but this needs to be fixed. I will send a new version.

**Review by fmuyassarov — APPROVED** [review]
LGTM

**Pull request merged**